### PR TITLE
Tainting updates

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/core/manager.c [hunk 1]
```diff
@@ -3901,12 +3901,6 @@ char *manager_taint_string(Manager *m) {
         if (r < 0 || !PATH_IN_SET(destination, "../run", "/run"))
                 e = stpcpy(e, "var-run-bad:");
 
-        if (!streq(NOBODY_USER_NAME, "nobody"))
-                e = stpcpy(e, "weird-nobody-user:");
-
-        if (!streq(NOBODY_GROUP_NAME, "nobody"))
-                e = stpcpy(e, "weird-nobody-group:");
-
```
Comment (poettering) [inline src/core/manager.c:3909]: hmpf. but ok. but you need to update the size estimation further above too.

Also, could you add a comment here that reminds people that future newly added taints should only be added for non-compile-time things?

**Comment by keszybz** [comment]
Updated.

**Comment by poettering** [comment]
OK, let's merge this, looks good to me, and two CIs already passed and this is unlikely going to break the others, and the others already ran partially with a slightly different earlier version.

**Comment by mbiebl** [comment]
Changing an established group like nogroup in a distro like Debian is very very painful due to the vast amount of heritage attached. So we really need a very good, very compelling reason why group nogroup should be renamed. So far I haven't read a convincing technical argument.

**Pull request merged**